### ISSUE TITLE
Questions about testing on human3.6M

### ISSUE THREAD

**Issue by booker-max** [mks0601/I2L-MeshNet_RELEASE#63]
Hello, I would like to ask if the pose_coord_gt_h36m you provided on the Human3.6M test set was obtained by using Smplify-x to estimate the SMPL parameters?
![image](https://user-images.githubusercontent.com/74257220/114333648-daca0780-9b59-11eb-9c5d-8c54b514188a.png)

**Comment by mks0601** [comment]
No. As you can see in [here](https://github.com/mks0601/I2L-MeshNet_RELEASE/blob/5a202da5e7719bb9977bc4c7b74a4a925de483c6/data/Human36M/Human36M.py#L141), `joint_cam` is from Human3.6M annotation file, provided by the dataset capture.

**Comment by booker-max** [comment]
ok，Thank you!

**Comment by booker-max** [comment]
I’m sorry to bother you again. I saw a sentence in your paper "Alternatively, we obtain groundtruth SMPL parameters by applying SMPLify-X on the groundtruth 3D joint coordinates of Human3.6M dataset." time,

1. Did you use groundtruth?

2. How are groundtruth 3D joint coordinates used when calculating smpl parameters?

3. Can you share your code for calculating smpl parameters, thank you very much.
